tunedev/Banka
# #165634314 Adds api documentation

#### What does this PR do?
This pr adds apps API documentation

#### Description of Task to be completed?
- Implement feedback gotten during apps review by LFA
- Add documentation router

#### How should this be manually tested?
> - Clone this repo and cd into its root folder
> - Run `npm install` and `npm start`
> - Visit this URL http://localhost:3000/api-docs
> - The API documentation should be displayed, switch server to localhost
> - and try out all routes

#### Any background context you want to provide?
NA
#### What are the relevant pivotal tracker stories?
[#165634314](https://www.pivotaltracker.com/story/show/165634314)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/33371467/56865613-1187d100-69c8-11e9-947b-8385f79a7cec.png)

#### Questions:
NA
